### PR TITLE
Feature 1806/add user from hr page

### PR DESCRIPTION
**Pull request by MatthewBackes**
-Allows User to click "Add Onboardee" button to bring up a modal to create a user
-Text fields for First Name, Last Name, Email
-Autocomplete Fields for Position, Hire Type, PDL/Manager (options are dummy data for now)
-Input fields for Offer Letter and Employment agreement (only allows for PDFS to be uploaded).

### CODE REVIEW

**Review by danie1r — APPROVED** [review]

**Review by BenVuong — APPROVED** [review]

File: web-ui/src/pages/OnboardProgressPage.jsx [hunk 1]
```diff
@@ -100,7 +235,7 @@ export default function OnboardProgressPage() {
           checkboxSelection
           disableSelectionOnClick
         />
-      </Box>
+      </Box >
```
Comment (S78901) [inline web-ui/src/pages/OnboardProgressPage.jsx:238]: Extra space.

File: web-ui/src/pages/OnboardProgressPage.jsx [hunk 2]
```diff
@@ -110,6 +245,6 @@ export default function OnboardProgressPage() {
           rowsPerPageOptions={[10]}
         />
       </Box>
-    </div>
+    </div >
```
Comment (S78901) [inline web-ui/src/pages/OnboardProgressPage.jsx:248]: Also extra space.

File: web-ui/src/pages/OnboardProgressPage.jsx [hunk 3]
```diff
@@ -89,9 +127,106 @@ export default function OnboardProgressPage() {
...
+            <Grid container>
+              <Grid item xs={6} style={{display: "flex", justifyContent: "flex-start"}}>
+              <Button variant="contained" onClick={handleClose} >Cancel</Button>
+              </Grid>
+              <Grid item xs={6} style={{display: "flex", justifyContent: "flex-end"}}>
+                <Button variant="contained">Submit</Button>
+              </Grid>
+            </Grid>
+          </Box>
+
```
Comment (S78901) [inline web-ui/src/pages/OnboardProgressPage.jsx:228]: Delete extra line.

**Review by S78901 — COMMENTED** [review]
This is looking pretty good, just a few minor fixes.

**Review by S78901 — APPROVED** [review]
Great work here.

**Review by BenVuong — APPROVED** [review]

**Review by danie1r — APPROVED** [review]

**Pull request merged**